### ISSUE TITLE
undefined moment in Library.js

### ISSUE THREAD

**Issue by AlexDWu** [exquisite-legume/epilog#102]
this occurred after reloading the client after #101
in library.js view line 69, story.moment[0].url is undefined.
The newly created story does not have any moments (it should have the newly created moment)

**Comment by AlexDWu** [comment]
New moment is not in database but new story is. Has something to do with JSON.parse on line 15 in momentController.js req.headers.tags was undefined 

Stack trace:
Failed to retrieve story by title:  [TypeError: Cannot set property 'userId' of undefined]
SyntaxError: Unexpected token u
    at Object.parse (native)
    at module.exports.add (/Users/alexanderwu/Code/HackReactor/epilog/server/moments/momentController.js:15:24)
    at Layer.handle [as handle_request](/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/route.js:131:13)
    at Route.dispatch (/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request](/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/index.js:277:22
    at Function.process_params (/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/index.js:330:12)
    at next (/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/index.js:271:10)
    at module.exports.authenticateToken (/Users/alexanderwu/Code/HackReactor/epilog/server/auth/authController.js:64:7)
    at Layer.handle [as handle_request](/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/index.js:312:13)
    at /Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/index.js:280:7
    at Function.process_params (/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/index.js:330:12)
    at next (/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/index.js:271:10)
    at serveStatic (/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/serve-static/index.js:74:16)
    at Layer.handle [as handle_request](/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/index.js:312:13)
    at /Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/index.js:280:7
    at Function.process_params (/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/index.js:330:12)
    at next (/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/express/lib/router/index.js:271:10)
    at jsonParser (/Users/alexanderwu/Code/HackReactor/epilog/server/node_modules/body-parser/lib/types/json.js:107:37)

**Comment by AlexDWu** [comment]
Line 59 in storyController is a problem if no stories are found
